Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
Ah okay.  I'll try to get some more tests written up just to clarify things in the morning and update the PR.